lnrpc/rpc.swagger.json

```diff
@@ -441,59 +429,25 @@
         "operationId": "CloseChannel",
         "responses": {
           "200": {
-            "description": "A successful response.(streaming responses)",
+            "description": "(streaming responses)",
             "schema": {
-              "$ref": "#/x-stream-definitions/lnrpcCloseStatusUpdate"
+              "$ref": "#/definitions/lnrpcCloseStatusUpdate"
             }
           }
         },
         "parameters": [
           {
             "name": "channel_point.funding_txid_str",
-            "description": "*\nHex-encoded string representing the byte-reversed hash of the funding\ntransaction.",
             "in": "path",
             "required": true,
             "type": "string"
           },
           {
             "name": "channel_point.output_index",
```

Comment:
There are a few desciption fields that are now gone. I suppose this is generated but should these fields remain?

Reply:
Not sure what's happening here, I'm just running `make rpc`, which should work out. Will investigate.

Reply:
Needed to reset my proto back to the right version, not sure how it got out of sync since I use `make rpc`, but all sorted after some good old fashioned manual intervention 😬